### PR TITLE
Contribution: Changing heart

### PR DESCRIPTION
**Pull request by Serpelic**

### CODE REVIEW

File: _CodingChallenges/134-heart-curve.md [hunk 1]
```diff
@@ -17,6 +17,19 @@ videos:
...
+    name: "Simon"
+    url: ""
+    url: "https://i.imgur.com/mXMyw9X.gifv"
+    source: "https://github.com/Serpelic/Processing_Projects/tree/master/Changing_heart"
+    
 ---
 
 Using a parametric equation from Wolfram Mathworld (and polar to cartesian coordinate transformation), I draw a heart curve in Processing (Java).
+
+
```
Comment (anuraghazra) [inline _CodingChallenges/134-heart-curve.md]: only one line

File: _CodingChallenges/134-heart-curve.md [hunk 2]
```diff
@@ -17,6 +17,19 @@ videos:
     url: "/Tutorials/9-additional-topics/9.20-polar-coordinates"
   - title: "Drawing with Fourier Transform and Epicycles"
     url: "/CodingChallenges/130.1-fourier-transform-drawing"
+    
+
```
Comment (anuraghazra) [inline _CodingChallenges/134-heart-curve.md]: remove this line

File: _CodingChallenges/134-heart-curve.md (hunk 1 above)
Comment (Serpelic) [reply _CodingChallenges/134-heart-curve.md]: How where what. Just a sec, I've never used github before.

Comment (Serpelic) [reply _CodingChallenges/134-heart-curve.md]: Done?

Comment (anuraghazra) [reply _CodingChallenges/134-heart-curve.md]: remove the extra line, you only need one line here. (yeah it also happened to me at first contrib)

File: _CodingChallenges/134-heart-curve.md (hunk 2 above)
Comment (anuraghazra) [reply _CodingChallenges/134-heart-curve.md]: i think you also need to remove last two lines

Comment (anuraghazra) [reply _CodingChallenges/134-heart-curve.md]: yeah, this is frustrating and confusing. but I said last two lines, not all the last lines (the rules are very strict)

**you can refer to other peoples contribution to see which is the correct syntax**

**Comment by anuraghazra** [comment]
@meiamsome  please check the commits and also this is the case where I should refer to https://github.com/CodingTrain/website/issues/1075

**Comment by gruselhaus** [comment]
❤️

**Pull request merged**